### PR TITLE
Added Hyperlane + LayerZero to Docs Site

### PR DESCRIPTION
**Pull request by jboetticher**
### Description

Added pages for Hyperlane and LayerZero under bridges

### Checklist

- [X] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [X] If images have been added, I have run the `compress-images.py` script to compress the images.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Comment by jboetticher** [comment]
Not sure why the build failed. Sounds like an mkdocs issue, since it didn't give many details

**Comment by eshaben** [comment]
The build for Kevin's PR failed too... After quickly looking into it, the OS system was bumped from Ubuntu 20.04.5 to 22.04.01 so that might have something to do with it. I'll look into it tomorrow

File: builders/integrations/bridges/hyperlane.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+![Hyperlane Technology Stack diagram](/images/builders/integrations/bridges/hyperlane/hyperlane-1.png)
+
+The Hyperlane APIs provide a rich suite for developing Web3 applications, ensuring that developers have the tools they need for building. With these tools and APIs, developers can use the Hyperlane protocol and its APIs to write dApps that can be easily deployed across all Hyperlane-connected ecosystems.
+
+--8<-- 'text/disclaimers/third-party-content-intro.md'
+
+## Getting Started {: #getting-started }
+
+There are a couple of resources to get you started building cross-chain applications with Axelar:
```
Comment (eshaben) [inline builders/integrations/bridges/hyperlane.md:23]:
```suggestion
There are a couple of resources to get you started building cross-chain applications with Hyperlane:
```

File: builders/integrations/bridges/hyperlane.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- **[Developer documentation](https://docs.hyperlane.xyz/hyperlane-docs-1/introduction/readme){target=_blank}** - for technical guides
+- **[Hyperlane Explorer](https://explorer.hyperlane.xyz/){target=_blank}** - to track cross-chain transfers
+
+## Contracts {: #contracts }
+
+See the list of Axelar contracts deployed to Moonbeam, and the networks connected to Moonbeam through Axelar.
```
Comment (eshaben) [inline builders/integrations/bridges/hyperlane.md:30]:
```suggestion
See the list of Hyperlane contracts deployed to Moonbeam, and the networks connected to Moonbeam through Hyperlane.
```

File: builders/integrations/bridges/hyperlane.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: Cross-Chain Via Hyperlane
+description: Learn how to bridge assets, plus other ways you can connect your Moonbeam dApp to assets and functions on multiple blockchains using Hyperlane protocol & APIs.
+---
+
+# Hyperlane Protocol
+
+![Hyperlane Moonbeam banner](/images/builders/integrations/bridges/hyperlane/hyperlane-banner.png)
+
+## Introduction {: #introduction }
```
Comment (eshaben) [inline builders/integrations/bridges/hyperlane.md:10]:
```suggestion
## Introduction {: #introduction }

```

File: builders/integrations/bridges/layerzero.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# LayerZero Protocol
+
+![LayerZero Moonbeam banner](/images/builders/integrations/bridges/layerzero/layerzero-banner.png)
+
+## Introduction {: #introduction }
+
+[LayerZero](https://layerzero.network/){target=_blank} delivers secure omnichain interoperability for Web3. It consists of an Oracle actor and a Relayer actor that work separately to provide secure messages from one chain to another. LayerZero's infrastructure enables dApp users to interact with any asset or application, on any connected chain, with one click.
+
+LayerZero is a transport layer that enables asset transfer with low-level communication primitives. dApps utilized LayerZero are known as User Applications, whose messages are relayed across chains by a Relayer, and whose messages are verified on the destination chain by an Oracle's block headers. Take a look at the tech stack diagram below and their [conceptual documentation](https://layerzero.gitbook.io/docs/faq/messaging-properties){target=_blank} for more details.
```
Comment (eshaben) [inline builders/integrations/bridges/layerzero.md:14]:
```suggestion
LayerZero is a transport layer that enables asset transfer with low-level communication primitives. DApps that utilize LayerZero are known as User Applications, whose messages are relayed across chains by a Relayer, and whose messages are verified on the destination chain by an Oracle's block headers. Take a look at the tech stack diagram below and their [conceptual documentation](https://layerzero.gitbook.io/docs/faq/messaging-properties){target=_blank} for more details.
```

File: builders/integrations/bridges/hyperlane.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+[Hyperlane](https://hyperlane.xyz){target=_blank} is a security-modular cross-chain communication protocol for Web3. Hyperlane enables dApp users to interact with any asset or application, on any connected chain, with one click. It supports general asset transfer as well as custom cross-chain messaging.  
+
+Using a method called [sovereign consensus](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/security/sovereign-consensus){target=_blank}, Hyperlane allows developers to configure the method by which messages are sent and validated across chains. Hyperlane is composed of validators, relayers, and watchtowers. Validators will watch for and confirm cross-chain messages. Relayers spend the gas to send messages across chains. Watchtowers perform checks to ensure that validators are good-faith actors, securing the protocol. Take a look at the tech stack diagram and their [protocol documentation](https://docs.hyperlane.xyz/hyperlane-docs-1/protocol/overview){target=_blank} for more details.
+
+![Hyperlane Technology Stack diagram](/images/builders/integrations/bridges/hyperlane/hyperlane-1.png)
```
Comment (eshaben) [inline builders/integrations/bridges/hyperlane.md:15]: Is there a better image we could use here? It's just really small on the docs site... I assume not, but figured I would check

**Review by eshaben — CHANGES_REQUESTED** [review]
Looks good!! Just need index page icons for both pages 🙂

File: builders/integrations/bridges/hyperlane.md (hunk 5 above)
Comment (jboetticher) [reply builders/integrations/bridges/hyperlane.md:15]: Unfortunately that's the only image that they've made. It's kind of wide, I agree

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**